### ISSUE TITLE
validation loss plot with customized task

### ISSUE THREAD

**Issue by Sujit27** [google-research/text-to-text-transfer-transformer#703]
I am using a customized task and training on translation dataset which has both training and validation data as tsv files. After training, I can see the resulting training loss plot on tensor board, but no plot for validation loss. How can I add it ?

BN_EN_DATA_PATH = {'train':'gs://my_bucket_train/train_bn_en.tsv','validation':'gs://my_bucket_train/dev_bn_en.tsv'}

t5.data.TaskRegistry.add(
    "fine_tune_bn_en",
    t5.data.TextLineTask,
    split_to_filepattern=BN_EN_DATA_PATH,
    text_preprocessor=[t5.data.preprocessors.parse_tsv],
    output_features=DEFAULT_OUTPUT_FEATURES,
    metric_fns=[t5.evaluation.metrics.bleu]
)

**Comment by craffel** [comment]
Hi, you need to run eval https://github.com/google-research/text-to-text-transfer-transformer#eval

**Comment by oathaha** [comment]
> Hi, you need to run eval https://github.com/google-research/text-to-text-transfer-transformer#eval

I saw that there are different metrics that I can use for validation set but I cannot find how to plot validation loss. Can you explain your answer in more detail?
